### PR TITLE
Journal changes additions

### PR DESCRIPTION
**Pull request by kingy04**
This update brings some layout/style improvements and fixes issues #38 and #40 as well as the unexpected reappearing of the popup issue. It also adds a slot for adding information on highlighted events.

### CODE REVIEW

File: src/components/shared/forms/croud-date-range.md [hunk 1]
```diff
@@ -7,3 +7,15 @@ Out of the box this component works as a singe date/date range combo selecter, w
...
+
+### Events
+Dates fed into the datepicker through the events array prop will be underlined in yellow, you can provide information on what these highlighted dates represent by using the events-info slot
+
+e.g. const yesterday = [moment().add(-1, 'day').toDate().toDateString()],
+
+    <croud-date-range :events="yesterday">
+        <div slot="events-info">
+            <semantic-divider class="hidden"/>
+            <p>Yesturday will always be highlighted</p>
```
Comment (matthewerskine) [inline src/components/shared/forms/croud-date-range.md]: Typo "yesterday"

Comment (kingy04) [reply src/components/shared/forms/croud-date-range.md]: That one gets me every time, thanks.

**Comment by BrockReece** [comment]
Are these changes on staging 4? If so, I still see #38

**Comment by kingy04** [comment]
@BrockReece  - I didn't publish them to S4 until now, please could you check again?

**Comment by kingy04** [comment]
Great thanks

**Comment by BrockReece** [comment]
I have tidied up some of these if statements, it looks ready to go, if we are happy?

**Comment by kingy04** [comment]
Looks good Brock, thanks.. I have made a last amendment to fix a comparison statement that wasnt quite right.. this is good to merge if we are all happy.. Thanks

**Comment by BrockReece** [comment]
Released in v1.7.3

**Pull request merged**